### PR TITLE
Get rid of existing cookie for storefront.

### PR DESCRIPTION
**Pull request by frankban**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6558/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6559/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6560/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6561/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6563/

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/6565/

File: jujugui/static/gui/src/app/app.js [hunk 1]
```diff
@@ -801,22 +801,25 @@ YUI.add('juju-gui', function(Y) {
...
-            // Set the model UUID to a value that communicates login handlers
-            // to not switch away from the deployment flow when the user will
-            // be required to authenticate later.
-            this.set('modelUUID', 'do-not-switch');
-            this.maskVisibility(false);
-            return;
-          }
+        if (!creds.areAvailable && !this.get('gisf')) {
+            // Credentials are not available and, not being in GISF mode, we
+            // don't support external or anonymous connections. Just show the
```
Comment (hatched) [inline jujugui/static/gui/src/app/app.js]: we do support external connections via USSO so this comment is a little misleading.

Comment (frankban) [inline jujugui/static/gui/src/app/app.js]: Yes in this case external is the one provided in juju_config.auth.

**Comment by frankban** [comment]
Thanks for the reviews! :+1:

**Comment by frankban** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**